client/src/app/components/Layout/Layout.jsx

```diff
@@ -41,8 +48,23 @@ export class Layout extends Component {
     loginUser();
   };
 
+  displayModeSwitch = () => {
+    const {
+      location: { pathname }
+    } = this.props;
+
+    return (
+      pathname.includes(`${Routes.COHORT}/`) ||
```

Comment:
and when the method will be named `isListView`  you can only check for one condition 

pathname.include(`Routes.LIST/`)   and whenever it returns true elements would be displayed, otherwise not